downloads/swan-lake-release-notes/swan-lake-2201.10.0/RELEASE_NOTE.md

```diff
@@ -113,19 +113,79 @@ To view bug fixes, see the [GitHub milestone for Swan Lake Update 10 (2201.10.0)
     ```
 
 #### OpenAPI tool
-
+-  Added support to generate a mock client for OAS if including the example details. To generate mock client, use `--mock` flag along with openAPI CLI client generation command.
+  ```
+  $ bal openapi -i <yml file> --mode client --mock
+  ```
+
+-  Provided the option to generate a single bal file for client/service. To generate single file, use `--single-file` flag along with openAPI CLI client/service generation command.
+  ```
+  $ bal openapi -i <yml file> --mode <client/service> --single-file
+  ```
+
+-  Added support to generate Ballerina service object type from given OAS. To generate a service contract object type file, use the `--with-service-contract` flag along with the OpenAPI CLI service generation command.
+  ```
+  $ bal openapi -i <yml file> --mode service --with-service-contract
+  ```
+
+-  Introduced example annotations in the OpenAPI tool. This feature will  enable rendering example schemas in the generated OpenAPI specification.
+ex:
+  1. Using Example annotation
+ ```ballerina
+    @openapi:Example {
+      value: {
+        id: 10,
+        name: "Jessica Smith"
+      }
+    }
+    type User record {
+      int id;
+      string name
+    }
+ ```
+  2. Using Examples annotation
+  ```ballerina
+    @openapi:Examples {
```

Comment:
Why were two different annotations introduced instead of one annotation that is allowed multiple times? Do the keys "Jessica", "Ron", etc. mean anything here?

Reply:
We introduced these two annotations, which represent the OAS properties themselves. In OAS, we have two different types of schema: `example` and `examples`. The keys have in relation to the OAS; if it is an `examples` schema, a key is always required for mapping it in the OAS schema.

Reply:
Ack, thanks.